app/digital/template.html

```diff
@@ -0,0 +1,281 @@
+            <tr>
+              <td width="100%" height="80" valign="top" style="text-align: center; vertical-align:middle;">
+              <!--[if gte mso 9]>
+              <v:rect xmlns:v="urn:schemas-microsoft-com:vml" fill="true" stroke="false" style="mso-width-percent:1000;height:80px; v-text-anchor:middle;">
+                <v:fill type="tile" src="http://s3.amazonaws.com/swu-filepicker/4E687TRe69Ld95IDWyEg_bg_top_02.jpg" color="#ffffff" />
+                <v:textbox inset="0,0,0,0">
+              <![endif]-->
+                <center>
+                  <table cellpadding="0" cellspacing="0" width="600" class="w320">
+                    <tr>
+                      <td class="pull-left mobile-header-padding-left" style="vertical-align: middle;">
+                        <a href=""><img width="150" height="30" src="https://abakus.no/185f9aa436cf7f5da598fd7e07700efd.png"></a>
+                      </td>
+                      <td class="pull-right mobile-header-padding-right" style="color: #4d4d4d;">
+                        <a href="https://www.facebook.com/AbakusNTNU"><img width="38" height="47" src="http://s3.amazonaws.com/swu-filepicker/LMPMj7JSRoCWypAvzaN3_social_09.gif"/></a>
+                      </td>
+                    </tr>
+                  </table>
+                </center>
+                <!--[if gte mso 9]>
+                </v:textbox>
+              </v:rect>
+              <![endif]-->
+              </td>
+            </tr>
+          </table>
+        </center>
+      </td>
+    </tr>
+    <tr>
+    <td align="center" valign="top" width="100%" style="background-color: #f7f7f7;" class="content-padding">
+      <center>
+        <table cellspacing="0" cellpadding="0" width="600" class="w320">
+          <tr>
+            <td class="header-lg">
+              VOTE
+            </td>
+          </tr>
+          <h2>Velkommen til Digital Genfors 2021!</h2>
+        </table>
+        <p>Hei! Vi ser du har meldt deg på Genfors 2021 på abakus.no. I år blir denne digital og vi har derfor laget en bruker i VOTE for deg. Hvis du ikke har vært på Genfors før og er usikker på hva dette betyr går det fint, da det vil bli forklart på Genfors. </p>
+        <hr/>
+        <p>Brukernavn: {{USERNAME}}</p>
+        <p>Passord: {{PASSWORD}}</p>
+        <p><i>Ditt passord har blitt generert av VOTE, og vi i Webkom vet ikke hva det er.</i></p>
+        <a href="https://vote.abakus.no" target="_blank">Logg inn her</a>
+      </center>
+    </td>
+  </tr>
+  <tr>
+    <td align="center" valign="top" width="100%" style="background-color: #ffffff;  border-top: 1px solid #e5e5e5;">
+      <center>
+        <table cellpadding="0" cellspacing="0" width="600" class="w320">
+          <tr>
+            <td style="padding: 25px 0 25px">
+              <strong>Abakus Linjeforening</strong><br />
+                <a mailto="webkom@abakus.no">webkom@abakus.no</a> <br />
+                <a href="{{ frontend_url }}">abakus.no</a> <br /><br />
```

Comment:
copy from lego?

Reply:
ooooooooo hehe :) 
I didn't bother making my own for VOTE. I know it's kinda stupid because it ties VOTE with Abakus, which is kinda bad since it's sometimes used by other people.

Reply:
(refered to the `{{ frontend_url }}` btw., since that one is lego-related :sweat_smile: )